src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,43 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+	private Set<Integer> numbers;
```

Comment:
```suggestion
class SetTest {
	Set<Integer> numbers;
```

JUnit5 부터는 클래스에 굳이 public 접근제어자를 설정해 줄 필요가 없어졌습니다~
그리고 private으로 테스트에 사용될 멤버를 감춰주셔도 되지만 그럴 필요가 없기 때문에 감춰주지 않으셔도 됩니다!

Reply:
넵, 몰랐던 부분을 알려주셔서 감사합니다.
다음부터는 생략해서 써야겠습니다.